tests/testthat/test-geom-ref-line.R

```diff
@@ -0,0 +1,5 @@
+context("geom-ref-line")
+
+test_that("can generate error messages", {
```

Comment:
I'd say "must supply at one argument".

Reply:
How does this sound? "at least one ref line position argument is supplied"